bin/acr-task-creation.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+export ACR_NAME=hmctspublic
+export GIT_PAT=$(az keyvault secret show --vault-name infra-vault-prod --name hmcts-github-apikey --query value -o tsv)
+
+az acr task create \
+  --registry $ACR_NAME \
+  --name jenkins-agent \
+  --context https://github.com/hmcts/cnp-gatling-image.git#master:jenkins-agent \
```

Comment:
the suffix is for the folder, we don't need java 11 anymore so I would delete the java 8 folder and move the contents of 11 to the root and remove #....